### ISSUE TITLE
tables colspan seems te be ignored

### ISSUE THREAD

**Issue by technetium** [2Abendsegler/GClh#682]
**Describe the bug**
I'm using colspan in a table to get a 12 column layout for my geocache page.
It's a multi langual page. I use 9 columns for the Dutch text, with a 3 column picture next to it.
Below are a 6 column English and a 6 column German text.
After GClh has loaded, the Dutch text will be camped into 6 columns and the images have 3 columns of white next to them.

**To Reproduce**

1. Goto https://coord.info/GC6WDEY
2. See the page as it is supposed to be
3. Wait until GClh had loaded
4. See that the Dutch text have 50% less space and the images 100% more

**Expected behavior**

The page layout is **not** changed by GClh

**Desktop (please complete the following information):**
 - OS: Windows 7
 - Browser: Both Firefox and Chrome (didn't tested it on others)
 - Version: To lazy to check them since to different browsers have the problem.

**Additional context**

Since the problems disappear when I turn of GClh I suspect GClh to be the problem.

**Comment by 2Abendsegler** [comment]
**Without GClh (970 Pixel):**
![without](https://user-images.githubusercontent.com/22332216/40557160-85b85618-604f-11e8-98cb-f16515eeb92a.JPG)


---
**With GClh (1100 Pixel):**
![with](https://user-images.githubusercontent.com/22332216/40557165-8874ce86-604f-11e8-9770-997d01df3900.JPG)

**Comment by 2Abendsegler** [comment]
Hi,
could it be that you set styles "max-width: 670px" in your listing?
If it is true please kill all these styles in your listing and try again.
Please let us know if that helped or not.
You are welcome.
Cheers,
Frank

Explanation: GClh set by 1100 pixel ".span-17 {width: " + (1100 - 300) + "px !important;}" to make the listing wider. The table in the listing is also affected of this. Because there seems to be max-width statements below the table, the table gets a width of 670 pixels. The table is not as far as I have drawn the arrow but stops behind the picture.

**Comment by technetium** [comment]
Hello Frank,

Thank you for your quick response. The  "max-width: 670px" styles are injected by Groundspeak in my listing. The code I submit does not contain these styles. Should I write a write a greasemonkey script to remove them? ;-)

**Comment by 2Abendsegler** [comment]
It seems to be a general problem and we have to have a solution for all GClh users, so the GClh script have to be changed. 
Do you have an idea since when the problem occurs?

**Comment by 2Abendsegler** [comment]
@Ruko2010 
Ich komm hier nicht weiter. Vielleicht kannst du es dir mal anschauen.

Wenn ich die Codingzeile 
`css += ".span-17 {width: " + (new_width - 300) + "px !important;}";
`
erst nach 20 Sekunden ausführe, dann sieht alles wie folgt aus bei 1200 Pixel Breite aus dem GClh.
![unbenannt](https://user-images.githubusercontent.com/22332216/40569883-bd139876-6085-11e8-89ca-d483c4be42ce.JPG)
Das ist so dann auch korrekt. Der obere Text hat Breite 900. Das "Download" ebenfalls mit center. Und die Tabelle unterm "Download" wird in der Breite 670 Pixel dargestellt weil GS bei einigen Tabellenelementen ein max-width von 670px gesetzt hat.

Wenn ich die Codingzeile von oben normal ausführe, dann wird alles verzerrt wie in meinem ersten Posting. Scheinbar wird nach Ausführung der Codingzeile an der Tabelle noch mal was geändert. Meines Erachtens aber nicht von uns. Für mich sieht es so aus, als ob GS nachträglich hier Änderungen an der Tabelle vornimmt. Beispielsweise enthält der table tag anschließend ein width: 670px.

Ich blick nicht wirklich durch und ich kann das eigentlich auch nicht glauben, weil das bedeuten würde, dass GS mit uns oder anderen Scripten rechnet.

Ich klink mich jetzt aus. Hab keine weiteren Ideen mehr.

**Comment by 2Abendsegler** [comment]
@Ruko2010 
Hab noch was geändert im Posting zuvor.

**Comment by Ruko2010** [comment]
@technetium are you sure that this is a gclh issue? If I disable it completely, the Listing looks like this on my Mac with Chrome:

![image](https://user-images.githubusercontent.com/2239440/40600267-180b7696-6251-11e8-804c-1da11d252a4e.png)

**Comment by Ruko2010** [comment]
Same issue on Safari (also with disabled GClh!):
![image](https://user-images.githubusercontent.com/2239440/40600381-78d63498-6251-11e8-8243-71276cbef8d2.png)

It seems like only Firefox is rendering the table correct, so maybe you should rewrite your table (or use a "div"-layout)?

**Comment by technetium** [comment]
I've checked for myself:
On a mac without GClh: Firefox: O.K. Safari and Chrome: 2 Column layout.

First I suspected the junk Groundspeak inc. is injecting in my code was to blame. Then I discovered the problems disappeared when I turned off CGlh so that extension became suspicious. Now I see the same problems on a Mac, It's probably back to Groundspeak inc. 

p.s. The code I submit to Groundspeak inc. is can be found at: https://gist.github.com/technetium/cbb08fa961a15f59ba3680461d4923d5#file-rewind-html this might give you an impression about what gets changed/injected.
